test/plugin/index.spec.js

```diff
@@ -1,10 +1,49 @@
-"use strict";
+'use strict';
 
-const Plugin = require("../../plugin");
+const Plugin = require('../../plugin');
+let plugin;
+let pluginWithOptions;
+const options = { port: 3000, host: '111.0.2.3' };
 
-describe("plugin", () => {
-  it("can create a new plugin", () => {
-    const plugin = new Plugin();
+beforeEach(() => {
```

Comment:
NIT: Scope the `plugin` + other vars _in_ the `describe` as well as this `beforeEach`. That way, we compartmentalize things better if someone greps.